README.md

```diff
@@ -307,20 +313,186 @@ IRIDA Next will attempt to autopopulate the `organism` column in the samplesheet
+## 3.3. Referencing external data
+
+The other type of parameter is one which can be used to reference external data (e.g., databases or other external files required to run the pipeline). These should be defined as a `string` type and use the **format** keyword to further refine how to interpret the type (see the [nf-schema format documentation][nf-schema format]).
+
+For example, in the [speciesabundance][] pipeline.
+
+```json
+"database": {
+    "type": "string",
+    "pattern": "^\\S+$",
+    "exists": true,
+    "format": "directory-path",
+    "description": "Path to database containing both the Kraken2 and Bracken database files (do not use symlinks)"
 }
 ```
 
-The full set of parameters can be found in <https://github.com/phac-nml/iridanext-example-nf/blob/main/nextflow.config>.
+The above specifies that the value of the parameter `--database` is a string which is interpreted as a `directory-path`.
+
+* This could be a **local path**: `--database /path/to/kraken_bracken_database/`
+* Or, this could be a **URI** specifying some resource. For example, using [Azure Blob Storage][]: `--database az://location/of/kraken_braken_database/`.
+
+When the pipeline is loaded within [IRIDA Next][irida-next], this type of parameter is rendered as a text box which can be used to fill in the path or URI to external data:
+
+![species-abundance-databases.png][]
+
+In order to configured a selectable list of external data in IRIDA Next, you can refer to the following section.
+
+## 3.4. Pipelines and parameters in IRIDA Next
+
+### 3.4.1. Loading pipelines
+
+Pipelines are registered within IRIDA Next by adding to a [pipelines.json][irida-next-pipelines] file. For example:
+
+**pipelines.json**
+```json
+{
+  "url": "https://github.com/phac-nml/speciesabundance",
+  "name": "phac-nml/speciesabundance",
+  "description": "IRIDA Next speciesabundance",
+  "versions": [
+    {
+      "name": "2.2.0"
+    }
+  ]
+}
+```
+
+When IRIDA Next is started, it will read the `pipelines.json` file, download a local copy of the pipeline `nextflow_schema.json` and `assets/schema_input.json` files within the specified repository and version (tag or branch) of a pipeline, and use these JSON Schema files to render a user interface for the pipeline.
+
+![irida-next-species-abundance.png][]
+
+### 3.4.2. Overriding parameters
+
+In addition to registering pipelines, the `pipelines.json` file can be used to override any of the Nextflow JSON Schema elements for a particular pipeline (e.g., default values, hidding parameters from IRIDA Next). For more information on this, please refer to the [IRIDA Next Schema Overrides documentation][irida-next-schema-overrides].
+
+One particular scenario that is well-suited for configuring a drop-down list of configured databases/external data in IRIDA Next is by overriding the parameter in the IRIDA Next `pipelins.json` file to switch to an enumerated type containing a list of pre-configured databases and labels to display in IRIDA Next. 
```

Comment:
A bit tough to read: Maybe "Overriding a database/external data parameter to an enumerated type in the IRIDA Next `pipelines.json` can be a way to set up a drop-down list of pre-configured databases and labels to display in the IRIDA Next interface."

Reply:
Thanks for the suggestions. I've added it in [504bd95](https://github.com/phac-nml/pipeline-standards/pull/28/commits/504bd9518329359d2c79d238abb9381888c9841c)